### PR TITLE
BZ1950635: updating etcd and temporary control plane info

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1950635

4.5+

Preview: https://deploy-preview-33109--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-installation.html#installation-process_architecture-installation

### CODE REVIEW

**Comment by retroflexer** [comment]
Thanks. LGTM.

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL or send this to a better reviewer?

**Comment by jianlinliu** [comment]
LGTM

File: modules/installation-process.adoc [hunk 1]
```diff
@@ -82,11 +82,11 @@ The Ignition config files that the installation program generates contain certif
...
-. The master machines fetch the remote resources from the bootstrap machine and finish booting. (Requires manual intervention if you provision the infrastructure)
-. The master machines use the bootstrap machine to form an etcd cluster.
-. The bootstrap machine starts a temporary Kubernetes control plane using the new etcd cluster.
-. The temporary control plane schedules the production control plane to the master machines.
+. The bootstrap machine boots and starts hosting the remote resources required for the control plane machines to boot. (Requires manual intervention if you provision the infrastructure)
+. The bootstrap machine starts a single-node etcd cluster and a temporary Kubernetes control plane.
+. The control plane machines fetch the remote resources from the bootstrap machine and finish booting. (Requires manual intervention if you provision the infrastructure)
+. The temporary control plane schedules the production control plane to the production control plane machines.
+. The Cluster Version Operator (CVO) comes online and installs the etcd operator. The etcd Operator scales up etcd on all control plane nodes.
```
Comment (bergerhoffer) [inline modules/installation-process.adoc:89]: operator -> Operator

**Review by bergerhoffer — COMMENTED** [review]
Just one thing!

**Pull request merged**